### ISSUE TITLE
OW desktop: ITALOccult feed at configurations menu in other languages than English

### ISSUE THREAD

**Issue by oliverkloes** [IOTA-NA/OW-Issues#1]
Today I downloaded the latest desktop version of OW and found the new feed ITALOccult.

At the configuration menu in English everything is o.k. but in other languages (I checked German, French and Castellano) the RIO feed was replaced with the text for the ITALOccult feed and the entry for the ITALOccult which follows has no description. The feed itself works. See screen shot attached of the German version. Is it easy to fix? 
Thanks,
Oliver

**Comment by hpavlov** [comment]
Hi Oliver,

you are correct some of texts have not been translated and I missed to add the default English text of the ITALOccult feed for the other languages. I will make texts available for translation probably via GitHub and will then inform current translators so they can do the missing parts.

**Comment by oliverkloes** [comment]
Hi Hristo,
many thanks for providing the translation files.
I just made my translations today and revised some of my previous translations.

I found some missing parts in the German translation file:

Feed description missing:
IBEROC, Lunar Occultations, NEA

Add-Ins:
Show Tangra Reports
Configuration C2A Add-In: missing some translations
Lunar Occultations
IOTA Reporting
Guide Add-In: some missing translations

Configuration:
Google maps:
Compatibility mode

Translation of the Text of agreement for RIO /Lucky Star feeds (I can't check this, but I think it was in English only)

By the way: I'M happy to provide any translation into German for OW Mobile as well.

Cheers,
Oliver
